README.md

```diff
@@ -71,6 +71,45 @@ Then the following data is inserted in the store:
         as:deleted "thetimeofdelete"^^xsd:dateTime  .
 ```
 
+## Extending Forms
+
+### Predicates and Types
+
+Use the same model as regular forms, with the following differences:
+-   don't use the `sh:group` predicate to denote connections to the owning form (or its sections/listings). Instead use `ext:extendsGroup`
+-   don't use the `form:includes` predicate to denote that a field is part of a form. Instead use `ext:extendsForm`
+
+An extension is represented by 
+
+    <extensionUri> a form:Extension ;
+                   form:includes <fieldUri> ;
+                   form:extends <formUri> ;
+                   mu:uuid "b5a86f3a-aac8-4911-a3fb-37f9f194b58e" .
+
+### Order
+
+We want fields to be added to the original form in a specific order, intermixed with the existing fields (before, after, between). Unfortunately, the order property is an integer and loses float/double precision if it is added to it. Therefore we should leave gaps of e.g. 100 between the order properties of our forms.
```

Comment:
Haha dit voelt echt als programmeren in BASIC, maar acceptable I guess, in mijn hoofd lijkt een extensie eerder iets dat we op het einde plaatsen, omdat misschien ook niet iedereen toegang heeft tot de form.ttl van de oorspronkelijke form, maar ik veronderstel dat het niet kwaad kan dit te supporten?

Reply:
yeah we will need to i'm afraid and I agree that it's not pretty :( 
people will be adding e.g. personal phone number and will want it next to the official one :/